envoy.bazelrc

```diff
@@ -42,6 +45,9 @@ build --action_env=CXX
 build --action_env=LLVM_CONFIG
 build --action_env=PATH
 
+# Disable ICU linking for googleurl.
+build --@com_googlesource_googleurl//build_config:system_icu=0
+
```

Comment:
What could the the benefit of disabling ICU linking compared to installing libicu-dev package in builder docker?

Reply:
No idea, I just synced `envoy.bazelrc` to the upstream Envoy `.bazelrc`. We keep our own settings at `.bazelrc` that first includes `envoy.bazelrc`.